build/base.sh

```diff
@@ -58,7 +58,11 @@ BASE_OBJDIR="$(make -C${SRCDIR}/release -V .OBJDIR)"
 setup_stage "${BASE_OBJDIR}/${BASE_DISTDIR}"
 
 # remove older object archives, too
-BASE_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/base.txz
+if [ ${PRODUCT_HOST} != ${PRODUCT_ARCH} -a ${SRCREVISION} != 11.2 ]; then
+	BASE_OBJ=$(make -C/usr/obj${SRCDIR}/${PRODUCT_TARGET}.${PRODUCT_ARCH}/release -V .OBJDIR)/base.txz
+else
+	BASE_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/base.txz
```

Comment:
I wonder if passing ${MAKE_ARGS} here is enough to fix this? Also we don't need the 11.2 workaround anymore

Reply:
Finally :))
i just started working again on arm images from stretch.
i will update this issue as soon i got there.